Core/HLE/sceKernelModule.cpp

```diff
@@ -1697,27 +1699,19 @@ static u32 sceKernelLoadModule(const char *name, u32 flags, u32 optionAddr)
 	if (!module) {
 		if (magic == 0x46535000) {
 			ERROR_LOG(LOADER, "Game tried to load an SFO as a module. Go figure? Magic = %08x", magic);
-			return -1;
+			// TODO: What's actually going on here?
```

Comment:
This is Tekken 6 and Soul Calibur being silly and loading all files in a directory as modules, including the SFO file that happens to sit around there, IIRC.